### PR TITLE
FEATURE: Bash Install Script

### PR DESCRIPTION
**Pull request by roachmd**
Feature: Created standard bash install script to deploy on-premise instances
#59

### CODE REVIEW

File: install.sh [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+  echo "-- Thank you.."
+  exit 0
+else
+  echo "....Starting Install"
+fi
+
+# Checking for Virtual Env
+echo "-- Checking for virtualenv "
+virtualenv=`which virtualenv`
+if [ ${#virtualenv} -eq 0 ]; then
```
Comment (johngriebel) [inline install.sh]: Might want to make sure it's a virtualenv that supports python 3.6

Comment (roachmd) [reply install.sh]: Ansible at this time is not 3.6 friendly.

File: install.sh [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+  echo "-- virtualenv not found."
+  echo "-- please install virtualenv"
+  exit 0
+else
+  echo "Python Virtual env found."
+  virtualenv --version
+fi
+
+# Install virtual env and Ansible
+echo "-- Installing python virtual env to venv"
```
Comment (jgwerner) [inline install.sh]: I think this would be `activating` virtualenv.

File: install.sh [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+else
+  echo "....Starting Install"
+fi
+
+# Checking for Virtual Env
+echo "-- Checking for virtualenv "
+virtualenv=`which virtualenv`
+if [ ${#virtualenv} -eq 0 ]; then
+  echo "-- virtualenv not found."
+  echo "-- please install virtualenv"
```
Comment (jgwerner) [inline install.sh]: Maybe we could echo what command is used to install virtualenv.

File: install.sh [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+  exit 0
+else
+  echo "Python Virtual env found."
+  virtualenv --version
+fi
+
+# Install virtual env and Ansible
+echo "-- Installing python virtual env to venv"
+virtualenv venv
```
Comment (jgwerner) [inline install.sh]: I think we should probably force python2 here. Ansible breaks with python3: `source -p python2 venv`.

File: install.sh (hunk 1 above)
Comment (johngriebel) [reply install.sh]: Ah, my bad. Was still in app-backend mode. Nevermind then!

**Pull request merged**